DataFormats/MuonReco/interface/Muon.h

```diff
@@ -175,15 +175,17 @@ namespace reco {
 
 
     /// define arbitration schemes
-    enum ArbitrationType { NoArbitration, SegmentArbitration, SegmentAndTrackArbitration, SegmentAndTrackArbitrationCleaned, RPCHitAndTrackArbitration };
+    enum ArbitrationType { NoArbitration, SegmentArbitration, SegmentAndTrackArbitration, SegmentAndTrackArbitrationCleaned, RPCHitAndTrackArbitration, ME0HitAndTrackArbitration };
```

Comment:
Why "Hit and Track"
 if I'm not mistaken, the objects used in matching are segments